### PR TITLE
[#124104671]Change contact link to email in footer

### PR DESCRIPTION
**Pull request by risicle**
I also took the liberty to fix the indentation of this file while I was here, I hope Sir doesn't mind.

Story https://www.pivotaltracker.com/story/show/124104671

### CODE REVIEW

**Comment by idavidmcdonald** [comment]
The indentation for this template still looks a little inconsistent for me, a mix of 2 space and 4 space?

**Comment by risicle** [comment]
Oh - yes - well caught. The indentation differences were bigger than I realized.

**Comment by risicle** [comment]
Think... that's better?

**Comment by risicle** [comment]
Same problem with the other one - will fix that too.

File: app/templates/_footer_categories.html [hunk 1]
```diff
@@ -33,34 +33,34 @@
...
-            <li>
-                <a href="https://www.gov.uk/guidance/digital-outcomes-and-specialists-suppliers-guide">Digital Outcomes and Specialists suppliers’ guide</a>
-            </li>
-        </ul>
-    </div>
-    <hr/>
+  </div>
+  <div class="footer-suppliers">
+    <h2>
+        Guidance
```
Comment (idavidmcdonald) [inline app/templates/_footer_categories.html]: One little 4 spacer might have slipped through here...

Comment (risicle) [inline app/templates/_footer_categories.html]: You're right. But I'm handing in my badge as the indentation police. It's not a fun job. Merging.

**Pull request merged**